jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/openshift/ImageStreamServiceTest.java

```diff
@@ -245,12 +245,12 @@ void appendImageStreamResource_whenMultipleTagsInImageStreamStatus_thenAddLatest
         assertThat(kubernetesList)
             .isNotNull()
             .extracting(DefaultKubernetesResourceList::getItems)
-            .asList()
+            .asInstanceOf(InstanceOfAssertFactories.list(ImageStream.class))
             .element(0)
             .asInstanceOf(InstanceOfAssertFactories.type(ImageStream.class))
             .hasFieldOrPropertyWithValue("metadata.name", "foo")
             .extracting("spec.tags")
-            .asList()
+            .asInstanceOf(InstanceOfAssertFactories.list(ImageStream.class))
```

Comment:
Shouldn't this be `TagReference` ?
```suggestion
            .asInstanceOf(InstanceOfAssertFactories.list(TagReference.class))
```

Reply:
^ @ShivamChavan01

Reply:
Sorry got busy with some interviews I'll do it right away